src/ObjLoading/Game/T6/BSP/BSPCreator.cpp

```diff
@@ -0,0 +1,276 @@
+#include "BSPCreator.h"
+
+#include "BSPUtil.h"
+
+#include <ufbx.h>
+
+namespace
+{
+    using namespace BSP;
```

Comment:
Usually you only do `using namespace` on global level since this technically embed one namespace into another which can get a little weird, especially if that namespace it is embedded into is named 😅 

adjusted in 08a8a5cd077ae217b1abeac4b0c8889f2b46d8b2